apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml

```diff
@@ -0,0 +1,808 @@
+      role: resident
+      checkErrorsOnlyInResponse: true
+      restMethod: post
+      validityCheckRequired: true
+      inputTemplate: injicertify/SunBirdC/GetCredentialSunBirdC/GetCredentialSunBirdC
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2SunBirdC_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateTokenSunBirdC_Valid_Smoke_sid_access_token$",
+        "format": "jwt_vc_json",
+      	"type": [{types: "VerifiableCredential"}, {types: "InsuranceCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+      	"proof_type": "jwt",
+        "proof_jwt": "$PROOF_JWT_2$"
+}'
+      output: '{
+      "error": "not_implemented"
+}'
+   InjiCertify_GetCredentialSunBirdC_IdpAccessToken_multiple_format_value:
+      endPoint: $INJICERTIFYINSURANCEBASEURL$/v1/certify/issuance/credential
+      uniqueIdentifier: TC_injicertify_credentialissuance_04      
+      description: Get sunbird VC when multiple values given as format    
+      role: resident
+      checkErrorsOnlyInResponse: true
+      restMethod: post
+      validityCheckRequired: true
+      inputTemplate: injicertify/SunBirdC/GetCredentialSunBirdC/GetCredentialSunBirdC
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2SunBirdC_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateTokenSunBirdC_Valid_Smoke_sid_access_token$",
+        "format": "ldp_vc, jwt_vc_json-ld",
+      	"type": [{types: "VerifiableCredential"}, {types: "InsuranceCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+      	"proof_type": "jwt",
+        "proof_jwt": "$PROOF_JWT_2$"
+}'
+      output: '{
+      "error": "unsupported_credential_format"
+}'
+   InjiCertify_GetCredentialSunBirdC_IdpAccessToken_onlyone_credentialtype_1:
+      endPoint: $INJICERTIFYINSURANCEBASEURL$/v1/certify/issuance/credential
+      uniqueIdentifier: TC_injicertify_credentialissuance_05      
+      description: Get sunbird VC when only one type is given for credential type     
+      role: resident
+      checkErrorsOnlyInResponse: true
+      restMethod: post
+      validityCheckRequired: true
+      inputTemplate: injicertify/SunBirdC/GetCredentialSunBirdC/GetCredentialSunBirdC
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2SunBirdC_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateTokenSunBirdC_Valid_Smoke_sid_access_token$",
+        "format": "ldp_vc",
+      	"type": [{types: "VerifiableCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+      	"proof_type": "jwt",
+        "proof_jwt": "$PROOF_JWT_2$"
+}'
+      output: '{
+      "error": "unsupported_credential_type"
+}'
+   InjiCertify_GetCredentialSunBirdC_IdpAccessToken_onlyone_credentialtype_2:
```

Comment:
is these are negative testcases, if yes please mention it in testcase name

Reply:
Updated